C++/include/algorithm/string/edit_distance.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+    Edit Distance Algorithm
+    ---------------------------------
+    This algorithm finds the minimum number of edits which are required 
+    to convert string1 to string2. Edit here means either insert,remove 
+    or replace. The algorithm implemented here uses dynamic programming.
+
+    Time Complexity
+    ---------------------------------
+    O(m*n)  
+    where-
+    m is length of string1
+    n is length of string2
+
+    Space Complexity
+    ---------------------------------
+    O(m*n)
+    where-
+    m is length of string1
+    n is length of string2
+*/
+
+
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <algorithm>
+
+/*
+    get_edit_distance
+    -----------------
+    This function takes arguments as two strings : str1,str2 . Then it finds 
+    the minimum number of edits required to change str1 to str2.
+*/
+
+int get_edit_distance (std::string str1, std::string str2) {
+    int m = str1.length(); //length of str1
+    int n = str2.length(); //length of str2
+    std :: vector <std :: vector <int> > dp (m + 1, std :: vector <int> (n + 1));
```

Comment:
Don't leave whitespace between `std` `::` `vector` `::` `<` etc. The following is better:
```c++
std::vector<std::vector<int>> dp...
```